repository-data/application/src/main/resources/hcm-config/namespaces/hee/listingPage.yaml

```diff
@@ -57,6 +59,14 @@ definitions:
             hipposysedit:primary: false
             hipposysedit:type: DynamicDropdown
             hipposysedit:validators: [required]
+          /imagelink:
```

Comment:
I think the node name should be `/heroImage` (instead of `/imagelink`)

Reply:
thanks for pushing this

Reply:
Fixed in 7a1eb1fdf601a33ec0e875008ea3a5df6d78c756